prov/efa/src/rxr/rxr_pkt_entry.h

```diff
@@ -172,7 +173,8 @@ struct rxr_pkt_entry {
 	 * @brief flags indicating the status of the packet entry
 	 * 
 	 * @details
-	 * Possisle flags include  #RXR_PKT_ENTRY_IN_USE #RXR_PKT_ENTRY_RNR_RETRANSMIT and #RXR_PKT_ENTRY_LOCAL_READ
+	 * Possisle flags include  #RXR_PKT_ENTRY_IN_USE #RXR_PKT_ENTRY_RNR_RETRANSMIT,
```

Comment:
nit: could you correct this (pre-existing) typo: "Possisle" -> "Possible"

Reply:
Another nit: The commit message mentions two functions "was removed", should be "were removed".

LGTM

Reply:
Thank you! fixed